contents/blog/posthog-vs-logrocket.md

```diff
@@ -0,0 +1,750 @@
+            <td>Only capture the sessions you want</td>
+        </tr>
+        <tr>
+            <td><strong>Export recordings</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Take recordings in/out of the platform</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+-   **Mobile app recordings:** PostHog does not currently offer recordings for mobile apps, but it is something we’re actively working on. Check [our roadmap](/roadmap) for more info!
+
+### Heatmaps and click tracking
+
+LogRocket’s heatmap and click tracking functionality exceeds PostHog’s and enables you to visualize more precisely how users interact with your site at the aggregate level. PostHog’s [toolbar](/manual/toolbar) focuses on visualizing clicks, and enabling users to [quickly track such events within product analytics](/tutorials/toolbar).
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+			<td className="w-3/12"></td>
+            <td><strong>LogRocket</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>Heatmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Visualize cursor movements on a page</td>
+        </tr>
+        <tr>
+            <td><strong>Clickmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Visualize where users click on a page</td>
+        </tr>
+        <tr>
+            <td><strong>Scrollmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Visualize how far down a page users scroll</td>
+        </tr>
+        <tr>
+            <td><strong>Eyetracking</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Visualize precisely where users are looking on a page</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+**Toolbar:** PostHog’s heatmap functionality (technically, clickmapping) is accessed via the [toolbar](/manual/toolbar). It enables you to visualize clicks as an overlay on a live page, but also enables you to quickly create events and actions which can be analyzed using PostHog’s analytics features.
```

Comment:
Not sure we should add the "techincally clickmapping" bit here. Feels a bit jarring.

Reply:
I'll defer to you and you can commit in a change if you want it. But I think it's important to be clear because what we call heatmapping in PostHog isn't actually heatmapping at all.